platformcomponents/desktop/date-picker.json

```diff
@@ -0,0 +1,60 @@
+{
+  "pickers": {
+    "comment": "Applied to date picker.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=51360%3A117328",
+    "datePicker": {
+      "month": {
+        "text": {
+          "#normal": "@theme-text-primary-normal"
+        }
+      },
+      "weekDay": {
+        "text": {
+          "#normal": "@theme-text-secondary-normal"
+        }
+      },
+      "date": {
+        "background":{
+          "#normal": "@theme-button-secondary-normal",
+          "#hovered": "@theme-button-secondary-hover",
+          "#pressed": "@theme-button-secondary-pressed",
+          "#disabled": "@theme-button-secondary-normal",
+          "#active": "@theme-background-accent-normal"
+        },
+        "text": {
+          "#normal": "@theme-text-primary-normal",
+          "#hovered": "@theme-text-primary-normal",
+          "#pressed": "@theme-text-primary-normal",
+          "#disabled": "@theme-text-primary-disabled",
+          "#active": "@theme-common-text-white"
+        },
+        "border": {
+          "#normal": "@theme-outline-button-normal"
+        },
+        "unreadBadge": {
+          "#normal": "@theme-background-accent-normal",
+          "#active": "@theme-common-text-white"
+        }
+      }
+    },
+    "dayPicker":{
+      "background":{
```

Comment:
create an additional layer called "day" and put "background", "text", "border" inside it. This is to make the addition of future element in this dayPicker easier (like a title, etc)